recipes/trackastra/meta.yaml

```
@@ -0,0 +1,65 @@
+{% set name = "trackastra" %}
+{% set version = "0.3.2" %}
```

```suggestion
{% set name = "trackastra" %}
{% set version = "0.3.2" %}
{% set python_min = "3.10" %}
```
Normally we don't need to set `python_min` in the recipe, but since our default value for `python_min` is 3.9, we need to raise to 3.10.